dotnet/src/SemanticKernel.MetaPackage/SemanticKernel.MetaPackage.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+  </PropertyGroup>
+  <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.SemanticKernel.MetaPackage</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.MetaPackage</RootNamespace>
```

Comment:
I don't think we actually call this "MetaPackage" (here or in file paths)... shouldn't this just be "Microsoft.SemanticKernel" and the other be under SK.Core?

Reply:
checking... last time I tried the build was failing...

Reply:
done